src/dataflow-types/build.rs

```diff
@@ -8,7 +8,7 @@
 // by the Apache License, Version 2.0.
 
 fn main() {
-    prost_build::Config::new()
+    tonic_build::configure()
```

Comment:
doesnt `tonic_build` just wrap `prost_build`? do we need this change?

Reply:
tonic build wraps prost build, but additionally, it understandes "service" definitions in proto files. So yes it's needed